nbs/01_lhs_parsing.ipynb

```diff
@@ -34,20 +34,21 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.                raise GraphRewriteException("Unexpected number of pattern sets in LHS.")
</pre><p>This is not possible based on how you defined the grammar, no?</p><p>If it is indeed not possible, no need to check it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>